utilities/backupable/backupable_db.cc

```diff
@@ -2789,27 +2790,27 @@ Status BackupEngineImpl::BackupMeta::LoadFromFile(
   }
 
   {
-    Status s = backup_meta_reader->GetStatus();
-    if (!s.ok()) {
-      return s;
+    IOStatus io_s = backup_meta_reader->GetStatus();
+    if (!io_s.ok()) {
+      return io_s;
     }
   }
 
   if (num_files != files.size()) {
-    return Status::Corruption(
+    return IOStatus::Corruption(
         "Inconsistent number of files or missing/incomplete header in " +
         meta_filename_);
   }
 
   files_.reserve(files.size());
   for (const auto& file_info : files) {
-    Status s = AddFile(file_info);
-    if (!s.ok()) {
```

Comment:
Can you change AddFile to return an IOStatus? Probably should also take const std::shared_ptr<>&

Reply:
I have thought about it before, but it seems AddFile does not actually do the IO. So, I'm kind of hesitating.